cli/core/go.mod

```diff
@@ -48,6 +48,7 @@ require (
 	k8s.io/api v0.24.2
 	k8s.io/apimachinery v0.24.2
 	k8s.io/client-go v0.24.2
+	sigs.k8s.io/cluster-api v1.2.7
```

Comment:
We should avoid adding this dependency on core-cli as we are just using `util.RandomString(4)` function. We should try to find some alternative.

Reply:
Removed the cluster-api dependency and added util function.